preface.asciidoc

```diff
@@ -123,7 +123,7 @@ We structure our explorations of architectural patterns around an example app,
 building it up chapter by chapter.  We use test-driven development (TDD) at
 work, so we tend to show listings of tests first, followed by implementation.
 If you're not used to working test-first, it may feel a little strange at
-the beginning, but we hope you'll soon get used to seeing code "being used,"
```

Comment:
this is not a typo I'm afraid.  It's the totally-crazy american convention of putting punctuation inside quotes, and O'Reilly insist on it :sob: